setup.cfg

```diff
@@ -1,10 +1,61 @@
+[metadata]
+name = pandas
+description = Powerful data structures for data analysis, time series, and statistics
+long_description = file: README.md
+long_description_content_type = text/markdown
```

Comment:
If using README as the long description is not desired, I could revert this.

Reply:
no that makes sense